cli/src/utils.ts

```diff
@@ -288,3 +288,10 @@ export const validateSubscriptionProtocols = ({
     }
   }
 };
+
+/**
+ * Synchronously writes to the console.
+ */
+export const writeToConsole = (message: string) => {
```

Comment:
Please express in the same that the operation is synchronous e.g. `writeToConsoleSync`